packages/phoenix-evals/src/phoenix/evals/functions/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .classify import llm_classify, run_evals, run_relevance_eval
```

Comment:
I'm not sure the `functions` directory is needed. Can we flatten this directory?